grants.yml

```diff
@@ -393,7 +395,9 @@
     - project:comm:thunderbird:releng:treescript:action:push
     - project:comm:thunderbird:releng:treescript:action:tagging
     - project:comm:thunderbird:releng:treescript:action:version_bump
-    - queue:create-task:low:scriptworker-prov-v1/tb-depsigning-mac-v1
+    - queue:create-task:low:scriptworker-prov-v1/tb-signing-mac-v1
```

Comment:
Likewise.

Reply:
I don't think this is fixed? This is a prod signing scope, granted to an L1 context.